ext/standard/array.c

```diff
@@ -2909,7 +2909,7 @@ PHP_FUNCTION(range)
 	}
 err:
 	if (err) {
-		zend_throw_error(NULL, "step exceeds the specified range");
+		zend_value_error("step exceeds the specified range");
```

Comment:
What do you think about also capitalizing this message in order to match the others?

Reply:
I agree with that but I don't know if I should do this in the same commit.

Reply:
I agree with that but I don't know if I should do this in the same commit.